github-connector/internal/eventparser/eventparser.go

```diff
@@ -8,9 +8,18 @@ import (
 )
 
 // EventParser adds proper structure to existing payload, so it can be consumed by Event-Service
+
+type eventparser struct {
+	EventParser
+}
+
 type EventParser interface {
-	GetEventRequestPayload(string) (EventRequestPayload, error)
-	GetEventRequestAsJSON(EventRequestPayload) []byte
+	GetEventRequestPayload(eventType, eventTypeVersion, eventID, sourceID string, data json.RawMessage) (EventRequestPayload, apperrors.AppError)
```

Comment:
We should consider encapsulating GetEventRequestPayload method - I wonder if it should be exported as it is, or should not be exported and be used internally by GetEventRequestAsJSON method which would be the public one.